docs/glossary.rst

```diff
@@ -0,0 +1,400 @@
+:ref:`project<glossary-project>` of an :ref:`organization<glossary-org>`.
+
+
+.. _glossary-profile:
+
+Profile
+-------
+
+In CrateDB Cloud's :ref:`CLI<glossary-cli>`, :ref:`Croud<glossary-croud>`,
+profiles are sets of configuration options. They define API :ref:`endpoints
+<glossary-endpoint>` and the desired output format of interaction with
+those endpoints. A Croud user can create multiple profiles and switch between
+them as desired.
+
+
+.. _glossary-project:
+
+Project
+-------
+
+A project is contained within an :ref:`organization<glossary-org>`. A project
+exists to contain any number of associated services (see :ref:`products
+<glossary-product>`) deployed in a particular :ref:`region<glossary-region>`
+for a specific organizational need. For example, an organization may use
+distinct projects to separate between development and production environments.
+
+A given organization can have any number of projects. Just as organizations
+have administrators and members, so projects have their own administrators and
+members. The two groups can but need not overlap. (For more on user roles in
+CrateDB Cloud and how to manage them, see our :ref:`reference for user roles
+<user-roles>`.)
+
+Each project has a name, an associated region, and a unique ID. For information
+on how to create a project, please refer to our `guide to creating a new
+project`_.
+
+
+.. _glossary-region:
+
+Region
+------
+
+A region in the sense used for CrateDB Cloud is a set of data centers (servers)
+grouped together on a geographic basis so as to not exceed a certain latency.
+CrateDB Cloud currently has three regions by default, but also offers the
+option to define custom regions.
+
+
+.. _glossary-saas:
+
+SaaS
+----
+
+SaaS stands for "Software-as-a-Service". It refers to a model where software is
+provided to customers on a :ref:`subscription<glossary-subscription>` basis,
+rather than a one-off payment, and is centrally hosted. CrateDB Cloud can be
+used as a service through its SaaS :ref:`offer<glossary-offer>` on |Microsoft
+Azure Marketplace|.
+
+
+.. _glossary-scale-unit:
+
+Scale unit
+----------
+
+The CrateDB Cloud subscription plans all come with a number of different scale
+units. Each scale unit represents an additional unit of the specific
+combination of hardware capacity that applies to that plan and is defined as
+one :ref:`DTU<glossary-DTU>` for that plan. Each scale unit represents
+one additional DTU, but not all plans have the same size of DTU or start at
+scale unit 1. For more information about subscription plans, refer also to our
+Documentation on :ref:`DTUs and Azure plans<azure-plans-dtus>`.
+
+
+.. _glossary-subscription:
+
+Subscription
+------------
+
+A subscription is - for the purposes of CrateDB Cloud - a container in which
+the CrateDB Cloud service is created and managed, supported by a particular
+Cloud provider. In other words, once a customer signs up for the CrateDB Cloud
```

Comment:
So far "Subscription" only applies to  Azure. Please note that.

It might be that AWS or GCP have a different term.